uberduck_ml_dev/trainer/base.py

```diff
@@ -150,9 +150,9 @@ def forward(self, model_output: List, target: List):
 
         mel_loss = mel_loss_batch.mean()
 
-        gate_loss_batch = nn.BCEWithLogitsLoss(pos_weight=None, reduce=False)(
-            gate_out, gate_target
-        ).mean(axis=[1])
+        gate_loss_batch = nn.BCEWithLogitsLoss(
+            pos_weight=self.pos_weight, reduce=False
```

Comment:
Bugfix, we weren't plumbing `self.pos_weight` through.

Reply:
ahhhhhhh.  good catch.  will pull this.